pkg/monitor/datapath_trace_test.go

```diff
@@ -6,7 +6,6 @@ package monitor
 import (
```

Comment:
Could you please add a brief summary on why we need to replace standard go benchmarks to gocheck to the commit message

Reply:
@brancz authored this commit so I'm not sure. I would guess because we already use gocheck for other tests through the file (although they are not benchmark). Seems like we're inconsistently using one or the other for benchmarks (e.g. [here](https://github.com/cilium/cilium/blob/c2b5e70535ef725b372b622be38b1467c6b7bd21/daemon/cmd/fqdn_test.go#L162) and [here](https://github.com/cilium/cilium/blob/c2b5e70535ef725b372b622be38b1467c6b7bd21/pkg/datapath/loader/loader_test.go#L246)).